### DISCUSSION TITLE
Error in api key registration for Anthropic

### DISCUSSION THREAD

**Discussion by shota1998** [langgenius/dify#10832] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

## what I want to know
How to solve this error.
What I should put in "API URL".

## error
> Error code: 404 - {'type': 'error', 'error': {'type': 'not_found_error', 'message': 'model: claude-instant-1.2'}}

<img width="1145" alt="image" src="https://github.com/user-attachments/assets/1d2d9d35-91f3-4273-8e9f-f4775ea96d84">

## process
1. create anthopic account.
2. resgister credit card info at anthropic.
3. register $10 credit at anthropic.
4. generate api key at anthropic.
5. put this api key at dify.
6. error occured at dify <----------- NOW

## PS
 values I entered in "API URL"
- https://api.anthropic.com/v1/messages/batches
- https://api.anthropic.com/v1/messages/
- https://api.anthropic.com/v1/
- https://api.anthropic.com/

**Comment by bathtimefish** [comment]
I got same probrem and can not get solves yet.

**Comment by bikeread** [comment]
I encountered the same issue. The debug log shows that it was caused by a cookie anomaly. After trying to log out and log back in, everything returned to normal.
